packages/twenty-front/src/modules/object-metadata/hooks/useCreateOneObjectMetadataItem.ts

```diff
@@ -34,8 +33,20 @@ export const useCreateOneObjectMetadataItem = () => {
       variables: {
         input: { object: input },
       },
-      awaitRefetchQueries: true,
-      refetchQueries: [getOperationName(FIND_MANY_OBJECT_METADATA_ITEMS) ?? ''],
+      optimisticResponse: {
+        createOneObject: {
+          ...input,
+          createdAt: new Date().toISOString(),
```

Comment:
we need to pass updatedAt, labelIdentifierFieldMetadataId, imageIdentifierFieldMetadataId otherwise the cache won't be properly updated (these error  messages were found in the console)